courses/utils_test.py

```diff
@@ -146,12 +170,37 @@ def test_generate_program_certificate_success(user, program):
 
     CourseRunCertificateFactory.create(user=user, course_run=course_run)
 
-    result = generate_program_certificate(user=user, program=program)
-    assert result[1] is True
-    assert isinstance(result[0], ProgramCertificate)
+    with patch.object(log, "info") as mock_log:
+        cert, created = generate_program_certificate(user=user, program=program)
+        mock_log.assert_called()
```

Comment:
It's basically never necessary to test that `log.info` was called.

```suggestion
    cert, created = generate_program_certificate(user=user, program=program)
```

In the future, if you want to patch `log`, use something like this:

```python
def test_something(mocker):
    mock_log = mocker.patch("courses.utils.log")
    do_error()
    mock_log.error.assert_called_once()
```